cogs/playerstats.py

```diff
@@ -136,41 +145,33 @@ def getPlayerStats(club_id):
 def getGoalsScored(club):
     df_topscorers = getPlayerStats(club)
     df_topscorers['goals'] = pd.to_numeric(df_topscorers['goals'])
-    df_topscorers = df_topscorers.sort_values(['goals'], ascending=False).head(5)
+    df_topscorers = df_topscorers.sort_values(['goals'], ascending=False).head(10)
```

Comment:
non blocking: may be this could be an argument that can be passed in from the discord command, with the default value set to some number (5)

Reply:
If we decide to ship this in another change, could you create an issue for it? Just in case we forget

Reply:
Will have to be a new issue as requires a bit of code shuffle for that, I left it out for that reason

This PR was primarily to align the goals and assists embeds